task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskDescriptionEditor.vue

```diff
@@ -19,24 +19,26 @@
     id="taskDescriptionId"
     :class="editorReady && 'active'"
     class="taskDescription">
-    <div
-      :data-text="placeholder"
-      :title="$t('tooltip.clickToEdit')"
-      contentEditable="true"
-      class="py-1 px-2 taskDescriptionToShow"
-      @click="showDescriptionEditor($event)"
-      v-html="inputVal ? urlVerify(inputVal) : inputVal">
-      {{ placeholder }}
-    </div>
-
     <exo-task-editor
       v-if="displayEditor"
       ref="richEditor"
       v-model="inputVal"
       :max-length="MESSAGE_MAX_LENGTH"
       :id="task.id"
+      @counterChanged="disableApply"
       :placeholder="$t('task.placeholder').replace('{0}', MESSAGE_MAX_LENGTH)" />
 
+    <v-btn
+      v-if="task.id!=null"
+      id="apply-btn"
+      depressed
+      outlined
+      class="btn mt-1 ml-auto d-flex"
+      :disabled="buttonOff"
+      @click="saveDescription(inputVal)"
+    >
+      Apply
```

Comment:
This line must be internationnalized. In addition, the string should be something like "Save"